app/controllers/passthrough_controller.rb

```diff
@@ -0,0 +1,18 @@
+class PassthroughController < ApplicationController
+  after_action :skip_authorization
+
+  def index
+    current_organisation = document_models.find { |model| model.organisations.include? current_user.organisation_content_id }
+
+    if current_user.gds_editor?
+      redirect_to "/aaib-reports"
+    elsif current_organisation
+      redirect_to("/" + current_organisation.slug) && return
+    else
+      redirect_to error_path
+    end
+  end
+
+  def error
```

Comment:
hmm. Maybe something a bit more specific. But if its rails magic then sticking with error might be best